tests/10-migration-backend-tests/testcafe/migrations/collection-01.csv

```diff
@@ -1,4 +1,4 @@
-unique_id,node_id,title,title_language,alternative_title,member_of,access_terms,contact_email,contact_name,collection_number,description,finding_aid
```

Comment:
Ok, so now the comments relating to "the degree to which our migration CSVs didn't pass muster"

Validation: extra/empty column